task7/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>JS課題7</title>
+  </head>
+  <body>
+    <div id="js-loading">
+      <ul id ="js-ul"></ul>
+    </div>
```

Comment:
Loading should be in the asynchronous process, so it is not optimal to have it in this location.

IMO
If you create the ul in JS and insert it into this js-parent, you can implement the above in JS.
```js
  　<div id="js-parent"></div>
```

Reply:
Thank you for your nice comment!
Thanks to that, I was able to think differently than before.